### PR TITLE
Added cantons of Angola

### PR DESCRIPTION
**Pull request by Zebedeu**
Added cantons of Angola.  All provinces or states of Angola have been added.

### CODE REVIEW

**Comment by claudiosanches** [comment]
Can you provide some source for the ISO codes?
I could just find this one: https://en.wikipedia.org/wiki/ISO_3166-2:AO
But is using 3 letters and not 2... Need a source just for future reference and make sure all is correct.
Thanks for your contribution.

**Comment by Zebedeu** [comment]
I found the following sources referring to the ISO of Angola and its respective provinces or states.

http://www.statoids.com/uao.html

http://194.132.104.143/GROUP/Standard/ACCodes.nsf/baf0033777518616c1256f43007bc4f2/0f6321291836c6bcc1256a11002665b9?OpenDocument

//en.wikipedia.org/wiki/ISO_3166-2:AO

**Comment by mikejolley** [comment]
@Zebedeu Are there any eCommerce stores base din Angola and do they use a dropdown of states or allow text input? What is more common?

**Comment by Zebedeu** [comment]
The shops and commerce in angola is a trend. And the stores use a dropdown menu for the states or provinces we designate #16059 
captura de ecra 2017-07-12 as 12 22 06
captura de ecra 2017-07-12 as 12 30 12

File: i18n/states/AO.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+global $states;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+$states['AO'] = array(
+		'BGO' => __( 'Bengo', 'woocommerce' ),
+	    'BLU' => __( 'Benguela', 'woocommerce' ),
+	    'BIE'  => __( 'Bié', 'woocommerce' ),
```
Comment (mikejolley) [inline i18n/states/AO.php]: The items here and below have extra spacing before the `=>`

Comment (Zebedeu) [inline i18n/states/AO.php]: Yes ... it was an indentation error. Will be corrected now.

**Comment by mikejolley** [comment]
@Zebedeu There are still 2 spaces before the `=>` on all the lines.

**Comment by mikejolley** [comment]
It was still very off in terms of indenting. Merged https://github.com/woocommerce/woocommerce/pull/16092

**Pull request closed without merging**